pkg/alertmanager/notify_hooks_notifier_test.go

```diff
@@ -206,7 +206,7 @@ func TestNotifyHooksNotifier(t *testing.T) {
 		_, err := f.notifier.Notify(ctx, makeAlert("foo")...)
 		require.NoError(t, err)
 
-		extraDataRaw, ok := f.upstream.ctxForTesting.Value(webhookV1.ExtraDataKey).([]json.RawMessage)
+		extraDataRaw, ok := f.upstream.ctxForTesting.Value(receivers.ExtraDataKey).([]json.RawMessage)
```

Comment:
I think this should still be webhookV1/ExtradDataKey

Reply:
Strange, there is no ExtraDataKey in receivers package.

Reply:
It was moved in the latest vendoring: https://github.com/grafana/mimir/pull/12888/files#diff-07aa7f27453b42ff416d71db891b00acf08e6740f82e0abb81ade98aa3cf5f2aR221

Reply:
Oh, I see.